Common/include/dual_grid_structure.inl

```diff
@@ -176,6 +182,36 @@ inline void CPoint::SetGridVel(su2double *val_gridvel) {
 		GridVel[iDim] = val_gridvel[iDim];
 }
 
+inline void CPoint::SetGridVel_Old(su2double *val_gridvel) { 
+	for (unsigned short iDim = 0; iDim < nDim; iDim++)
```

Comment:
Please change these tabs to spaces.

Reply:
Yep, went over all 'Files changed' and fixed all indentation mismatches I saw. 
Btw: In e.g. dual_grid_structure.hpp, most indentations are done using tabs and there are a lot of trailing whitespaces. Is it useful to change it in the whole file? Would be work of a few minutes but the latest git blame would be kindof misleading. I also noticed (which is probably) @WallyMaier in a branch `cleanup` just doing that cosmetic stuff. This is only a style(-guide) question.

Reply:
Since we are currently restructuring the files anyway, we should do that clean up when creating/moving new files. This way we avoid duplicate conflicts.

Reply:
Yep, please wait on the larger white space cleanup until we divide dual_grid_structure.hpp into files for each subclass.

Reply:
Hi all, I did start to create a clean up branch, referencing the the style guide.  I can continue with it (and skip the dual grid).  But I think with new version coming in, we should clean up files that are going to be restructured.